mirgecom/simutil.py

```diff
@@ -505,11 +507,17 @@ def compare_files_vtu(
             raise ValueError("Fidelity test failed: Mismatched data array sizes")
 
         # verify individual values w/in given tolerance
+        print(f"Field: {point_data2.GetArrayName(i)}", end=" ")
         for j in range(arr1.GetSize()):
-            if abs(arr1.GetValue(j) - arr2.GetValue(j)) > tolerance:
-                print("Tolerance:", tolerance)
-                raise ValueError("Fidelity test failed: Mismatched data array "
-                                 "values with given tolerance")
+            test_err = abs(arr1.GetValue(j) - arr2.GetValue(j))
+            if test_err > max_field_errors[i]:
+                max_field_errors[i] = test_err
+        print(f"Max Error: {max_field_errors[i]}")
+
+    violation = any([max_field_errors[i] > tolerance for i in range(nfields)])
+    if violation:
+        raise ValueError("Fidelity test failed: Mismatched data array "
+                                 "values with given tolerance. ")
 
```

Comment:
Would be good to include the tolerance value here.  Could be just:
f"Mismatched data array values {tolerance=}."

Reply:
Fixed! And the field names were Matthias' idea :)